src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -424,8 +540,8 @@ public class MyAttribute : System.Attribute
             Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
         }
 
-        [Fact]
-        public void TestFieldIsShadowed_ReferencedFromRegularLocalFunction()
+        [Theory, CombinatorialData]
+        public void TestFieldIsShadowed_ReferencedFromRegularLocalFunction([CombinatorialValues("always", "never")] string runNullableAnalysis)
```

Comment:
>[CombinatorialValues("always", "never")] string runNullableAnalysis [](http://example.com/codeflow?start=75&length=67)

Is this useful? For other new/modified tests as well.

Reply:
Not useful I think. I removed it